src/utility/VersionUtility.ts

```diff
@@ -48,25 +48,17 @@ export class VersionUtility {
     }
 
     // Returns the CLI version that this VSCode session expects
-    public static async getExpectedCliVersionAsync(context: vscode.ExtensionContext, experimentationService: IExperimentationService, packageJsonContent: object, logger: Logger): Promise<string> {
-        const bridgeEnvironment = EnvironmentUtility.getBridgeEnvironment(logger);
-
-        // This call will return the upcoming version for treatment group; control & everyone else will get the stable prod version. Undefined if the flight is currently inactive.
-        // TODO(ansoedal): Disable ExP permanently. For now this call will return undefined, as long as the treatment variable doesn't exist
-        const expBinariesVersion = experimentationService.getTreatmentVariable<string>(/*configId*/ `vscode`, /*name*/ `mindaroBinariesVersion-doesnotexist`);
-        logger.trace(`Call to ExP returned version '${expBinariesVersion}'`);
+    public static async getExpectedCliVersionAsync(packageJsonContent: object): Promise<string> {
+        const bridgeEnvironment = EnvironmentUtility.getBridgeEnvironment();
 
         // How we resolve the expected CLI version:
         //  1. Is the BRIDGE_CLI_VERSION environment variable set? Use that one.
         //  2. Is a flight in progress? Get the binaries version from ExP.
         //  3. Fall back onto whichever LKS version was specified in package.json. (For dev and staging, this is whichever version matches this extension. For Prod, this is a variable we specify in the Mindaro-Connect-VSCode pipeline.)
         const packageJsonVersion: string = packageJsonContent[`extensionMetadata`][`expectedCLIVersion`];
         const expectedCLIVersion = process.env.BRIDGE_CLI_VERSION != null ? process.env.BRIDGE_CLI_VERSION :
-            (expBinariesVersion != null && bridgeEnvironment === Environment.Production ? expBinariesVersion :
-            (process.env.BRIDGE_BUILD_PATH != null ? null :
```

Comment:
I thought I added a comment for this but can't find it anymore, so I hope I'm not duplicating things here. :)
By removing the ` process.env.BRIDGE_BUILD_PATH != null ? null :` part, I believe that this will prevent updates requiring both the CLI and the VS Code to be updated at the same time. That might be fine given that we're decorrelating things anyway here, but I just wanted to make it clear as a consequence.

Reply:
@daniv-msft you can refer the actual changes in another PR here - https://github.com/Azure/vscode-bridge-to-kubernetes/pull/9/files#r989107913, Can you please suggest there? 
Thanks Hari.